fsspec/implementations/tests/conftest.py

```diff
@@ -24,15 +25,23 @@ class MultiProtocolFileSystem(LocalFileSystem):
 
 @pytest.fixture(scope="function")
 def fs(request):
-    pyarrow_fs = pytest.importorskip("pyarrow.fs")
-    FileSystem = pyarrow_fs.FileSystem
     if request.param == "arrow":
+        pyarrow_fs = pytest.importorskip("pyarrow.fs")
+        FileSystem = pyarrow_fs.FileSystem
         fs = ArrowFSWrapper(FileSystem.from_uri("file:///")[0])
         return fs
     cls = FILESYSTEMS[request.param]
     return cls()
 
 
+@pytest.fixture(scope="function")
+def fscls(request):
+    try:
+        return fsspec.get_filesystem_class(request.param)
+    except ImportError:
+        pytest.skip(f"filesystem {request.param} not installed")
```

Comment:
I would have done 
```suggestion
    except ImportError as exc:
        pytest.skip(f"{request.param} not installed? Got {exc}")
```
since it is an assumption that `{request.param}` is the one causing ImportError but could be e.g. some module that one uses/relies upon

Reply:
It is only a skip message, so I'm not worried. Most accurate would be "{} could not be imported".